include/InterpUtils.h

```diff
@@ -3,9 +3,25 @@
 
 #include <vector>
 
-#include "Util.h"
+#include "Vectors.h"
 
 namespace ale {
+
+  enum RotationInterpolation {
+    SLERP, // Spherical interpolation
+    NLERP // Normalized linear interpolation
+  };
+
+  /// Interpolation enum for defining different methods of interpolation
+  enum interpolation {
```

Comment:
We should re-name this to PositionInterpolation to match the RotationInterpolation.